test/nexus-integration.py

```diff
@@ -166,33 +167,6 @@ def test_upload_package_with_local_identifier(self):
             )
             assert output['version']['version'] == version
 
-    def test_fail_to_upload_if_package_version_not_pep440_compliant(self):
```

Comment:
I am puzzled... in PR description you wrote:
> test/unittests.py ......... 
> 9 passed in 0.41s

so how was this one passing? if our assumption is correct... `subprocess.run(['python', 'setup.py', 'sdist']` should fail..

Reply:
? 
This is exactly the test that was failing... I don't understand your comment?

In the PR description it was about unit tests. Here it's integration tests

Reply:
> Sorry... ignore this one... this is the nexus integration test...

You should refresh :) I already apologized :)